### PR TITLE
Add aarch

### PR DESCRIPTION
**Pull request by pavelzw**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pavelzw** [comment]
Seems to work with https://github.com/conda-forge/conda-smithy/pull/1751

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,17 +19,18 @@ requirements:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - crossenv                            # [build_platform != target_platform]
-    - maturin >=0.14,<0.15                # [build_platform != target_platform]
+    - maturin >=1.2.1,<2                  # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
+    - rust 1.73.0.dev20230713
```
Comment (timkpaine) [inline recipe/meta.yaml:25]: this line probably needs to be reconciled with the line above it no?

Comment (0xbe7a) [reply recipe/meta.yaml]: i switched to using rust_compiler_version

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
One caveat: This will break automerge as that will rerender with the conda-smity release version.

**Comment by pavelzw** [comment]
Should we wait until https://github.com/conda-forge/conda-smithy/pull/1751 is released?

**Comment by 0xbe7a** [comment]
Sure, if the conda-smithy PR takes longer than a couple of weeks to merge we can also just setup the swap by hand and not have to manually rerender for eternity.

**Comment by jonashaag** [comment]
Nice work <3

**Pull request merged**